_episodes/l1-02-datastructures.md

```diff
@@ -154,22 +160,6 @@ replaced:
 TypeError: 'tuple' object does not support item assignment
 ```
 
-> ## Modifying a tuple vs modifying the element of a tuple
```

Comment:
I think this is important because immutability in Python can cause some confusion, but I agree that including it here might cause even more confusion. Maybe leave it for some sort of "extra" section at the end for them to review on their own?

Reply:
Thanks. My main thinking here was that it's a bit Python specific.